src/gflownet/algo/advantage_actor_critic.py

```diff
@@ -38,29 +35,25 @@ def __init__(
             A context.
         rng: np.random.RandomState
             rng used to take random actions
-        hps: Dict[str, Any]
-            Hyperparameter dictionary, see above for used keys.
-        max_len: int
-            If not None, ends trajectories of more than max_len steps.
-        max_nodes: int
-            If not None, ends trajectories of graphs with more than max_nodes steps (illegal action).
+        cfg: Config
+            The experiment configuration
```

Comment:
Since the type are in the arguments nowadays, there is no need to duplicate those in the docstring (any doc generation tool can read the type from the signature of the function/method).

Reply:
That would require quite some time to edit all your docstring but just sharing FYI moving forward.

Reply:
Actually I'm noticing my IDE is just generating the docstring with the type, so there's no extra typing involved, but yeah it does look redundant.